internal/resources/webform/webform.js

```diff
@@ -2126,11 +2127,11 @@ window.initGRPCForm = function(services, svcDescs, mtdDescs, invokeURI, metadata
             var name = $(cells[0]).val();
             var val = $(cells[1]).val();
             if (name !== "") {
-                metadataStr += ` -H "${name}: ${val}"`;
+                opts.push(`-H "${name}: ${val}"`);
             }
         }
 
-        gRPCurlTextArea.html(`<div>grpcurl -plaintext${metadataStr} -d '${requestDataJson}' ${window.target} ${service}.${method}</div>`);
+        gRPCurlTextArea.html(`<div>grpcurl ${opts.join(" ")} -d '${requestDataJson}' ${window.target} ${service}.${method}</div>`);
```

Comment:
```suggestion
        gRPCurlTextArea.html(`<div>grpcurl ${opts}${metadataStr} -d '${requestDataJson}' ${window.target} ${service}.${method}</div>`);
```
and revert the other changes